### PR TITLE
Feature/L3-410 Chinese translations fixes

### PR DESCRIPTION
**Pull request by Sisawat**
Closes #L3-410

**Summary**

- Adding overlooked email labels to i18n

**Change List (describe the changes made to the files)**

- `emaiLabel`
- `emailLinkLabel`
- `emailToggleLabel`
- New default text for `previewToggleLabel`

**Acceptance Test (how to verify the PR)**

- New `previewToggleLabel` should read "Additional Hours"
- Email labels should accept new i18n values

**Things to look for during review**

- [ ] Make sure all references to `phillips` class prefix is using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] All strings should be translatable.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue
- [ ] Make sure all commits messages follow convention and are appropriate for the changes

### CODE REVIEW

File: CHANGELOG.md
Comment (adietrich3074) [inline CHANGELOG.md:1]: This looks great. Will we be updating this file for all features and bug fixes going forward?

File: src/components/ViewingsList/utils.ts [hunk 1]
```diff
@@ -56,6 +60,9 @@ export const i18n = {
   viewingDatesLabel: '_viewingDatesLabel_',
   viewingHours1Label: '_viewingHours1Label_',
   viewingHours2Label: '_viewingHours2Label_',
+  emailLabel: '_emailLabel_',
```
Comment (adietrich3074) [inline src/components/ViewingsList/utils.ts:63]: For these i18n constants what is the purpose of the leading and trailing underscores?

**Review by adietrich3074 — APPROVED** [review]
Looks great to me. No issues. Just curiosity on my part 😆

File: src/components/ViewingsList/utils.ts (hunk 1 above)
Comment (davidicus) [reply src/components/ViewingsList/utils.ts:63]: To differentiate between the normal text and a "translated" text.

**Review by davidicus — APPROVED** [review]

**Pull request merged**